### ISSUE TITLE
Embed selected version of SDL2 library into AppImage package

### ISSUE THREAD

**Issue by pktiuk** [AntiMicroX/antimicrox#476]
Iw tould give us better control over libraries used by app. We don't have to rely on packeges used by the oldest Ubuntu version.

In current state `AppImage` version would miss key features introduced in: #416 and #464

**Comment by pktiuk** [comment]
@gombosg  
Is there any way to update one library in our flatpak runtime?  
To maintain proper support for newer features we will have to package the next antimicrox release with fresh SDL2

**Comment by gombosg** [comment]
Yes, but the we have to build SDL as a module and bundle with the main app. A bit more maintenance burden. Are there any features that only work with the latest SDL version?
https://docs.flatpak.org/en/latest/manifests.html#modules

**Comment by gombosg** [comment]
No problem, let me know if there are some packaging questions!
